### PR TITLE
Validate Files

### PR DESCRIPTION
**Pull request by rdvelazquez**
Validates the input alignment file using the [batchfile](https://github.com/veg/datamonkey-js/blob/master/lib/bfs/datareader.bf) and some of the [js code](https://github.com/veg/datamonkey-js/blob/master/app/models/msa.js) from datamonkey-js. As discussed with @stevenweaver, this does not implement all of the validation steps that are currently in place on datamonkey but does runs the `datareader.hbl` script. We can evaluate what other validation we may want later. (I'll open a JIRA issue for this so it doesn't get lost)

This PR also:
 - Adds `make -j 4 HYPHYMP` back into the make file (this appears to have been inadvertently removed when [troubleshooting the makefile](https://github.com/veg/hyphy-gui/commit/7b5c3b15de93b544a97f6b4588bbda2b1e9cfc98))
 - Adds the dependencies and imports to use the hyphy-vision css
 - Revises the `get_msa_path` file and `main.js` to copy the input file to `.data` when the job is submitted rather than when it is run.
 - Adds a placeholder for the branch selection component

### CODE REVIEW

File: src/bfs/shared/PostScript.bf [hunk 1]
```diff
@@ -0,0 +1,339 @@
+/*---------------------------------------------------------
```
Comment (stevenweaver) [inline src/bfs/shared/PostScript.bf]: PostScript?!

File: src/main.js [hunk 2]
```diff
@@ -18,9 +18,9 @@ let mainWindow
 function createWindow () {
 
   // Add the React DevTools (currently has path hard coded).
-  if (fs.existsSync('/Users/ryanvelazquez/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.2.0_0')) {
+  if (fs.existsSync('/Users/ryanvelazquez/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.2.1_0')) {
```
Comment (stevenweaver) [inline src/main.js]: hardcoded path

File: src/main.js [hunk 3]
```diff
@@ -18,9 +18,9 @@ let mainWindow
...
     BrowserWindow.addDevToolsExtension(
-      '/Users/ryanvelazquez/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.2.0_0'
+      '/Users/ryanvelazquez/Library/Application Support/Google/Chrome/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.2.1_0'
```
Comment (stevenweaver) [inline src/main.js]: hardcoded path

File: Makefile [hunk 4]
```diff
@@ -3,7 +3,7 @@
 hyphy:
 	echo "installing hyphy"
 	@if ! test -d ./.hyphy; then git clone http://github.com/veg/hyphy.git ./.hyphy/; fi
-	@cd ./.hyphy && git checkout master && git pull && git checkout 2.3.10 && cmake . && cd ../
+	@cd ./.hyphy && git checkout master && git pull && git checkout 2.3.10 && cmake . && make -j 4 HYPHYMP && cd ../
```
Comment (stevenweaver) [inline Makefile]: We probably do not want to specify a hardcoded number for the `-j` argument in `make`. At most, we would have to use an environment variable or else no `-j` argument.

File: src/bfs/shared/PostScript.bf (hunk 1 above)
Comment (rdvelazquez) [reply src/bfs/shared/PostScript.bf]: I'm not exactly sure what these are doing; I copied the datareader batch file and the shared folder from datamonkey-js

File: src/main.js (hunk 2 above)
Comment (rdvelazquez) [reply src/main.js]: I was able to get the react developer tooling to automatically install using https://github.com/MarshallOfSound/electron-devtools-installer

**Comment by stevenweaver** [comment]
@rdvelazquez it appears the travis build is failing and the new test isn't part of it yet.

**Comment by rdvelazquez** [comment]
@stevenweaver I included the path to HYPHYMP to make travis happy. Should be ready for your review now.

File: src/app.jsx [hunk 5]
```diff
@@ -5,6 +5,8 @@ const ipcRenderer = require('electron').ipcRenderer;
 const _ = require('underscore');
 const remote = require('electron').remote; // remote allows for using node modules within render process.
 const electronFs = remote.require('fs');
+const hyphyVision = require('hyphy-vision');
+require("style-loader!css-loader!sass-loader!hyphy-vision/dist/application.scss");
```
Comment (stevenweaver) [inline src/app.jsx]: Doesn't webpack take care of sass-loader? Do we need to change how we're deploying hyphy-vision's scss to help?

**Pull request merged**